src/reducers/issues.reducer.js

```diff
@@ -59,7 +59,7 @@ export default handleActions(
       set('issueComments', initialState.issueComments, state),
     [SET_ISSUES_PAGING]: (state, { payload }) => {
       const pageCount = get('header.last.page', payload);
-      return set('pageCount', +pageCount, state);
+      return set('pageCount', !pageCount ? null : +pageCount, state);
```

Comment:
why are you doing +pageCount? are you trying convert a string to a number?

Reply:
Yes

Reply:
It's better to use parseInt for parsing numbers. In general forcing types into other types shouldn't be done using operators, as it is too implicit and easy to miss.

parseInt is the recommend way to cast a number (you can also pass the radix as the second argument in case it's not a decimal base 10 number.

For other data types you should use the type constructor functions for example for a boolean value I would do: `Boolean(value)` or for a string: `String(value)`

Reply:
No problem, changed it.